autogen/writejson.m

```diff
@@ -0,0 +1,41 @@
+% WRITEJSON Take a json-compatible Matlab object and save it to a file
+%
+% WRITEJSON(data, filename)
+%
+% This function uses python to pretty-print the string before we save it to a file
+%
+% Inputs:
+%   data = json-compatible Matlab object
+%   filename = the output json file name
+
+% Copyright 2019 Flat Earth Inc, Montana State University
```

Comment:
Let's update the copyright to Audio Logic

Reply:
Done